suite-common/walletconnect/src/walletConnectConstants.ts

```diff
@@ -0,0 +1,10 @@
+export const WALLETCONNECT_MODULE = '@suite/walletconnect';
+
+export const PROJECT_ID = '203549d0480d0f24d994780f34889b03';
```

Comment:
we will need to create a new one and hand over ownership to adminkos later, right?

Reply:
Yes